spec/counters/workflow_counter_spec.rb

```diff
@@ -30,9 +30,13 @@
     end
 
     context "with workflow counts" do
+      let(:sws_params) do
+        { workflow: workflow, retired_at: DateTime.now, created_at: DateTime.now + 2.days }
```

Comment:
i explicitly want to test that it's not going to catch these based on created at dates